octoprint_filamentsensorsimplified/templates/filamentsensorsimplified_settings.jinja2

```diff
@@ -53,18 +56,28 @@
         </div>
     </div>
 
+    <h3>{{ _('Filament out action') }}</h3>
     <div class="control-group">
-        <label class="control-label" for="filamentsensorsimplified_settings_commandInput">{{ _('Command to send') }}</label>
+        <label class="control-label" for="filamentsensorsimplified_settings_commandInput">{{ _('Action') }}</label>
         <div class="controls">
-            <input id="filamentsensorsimplified_settings_commandInput" type="text" class="input-large" data-bind="value: settingsViewModel.settings.plugins.filamentsensorsimplified.g_code, disable:printing">
+            <select data-bind="value: settingsViewModel.settings.plugins.filamentsensorsimplified.cmd_action, disable:printing" required>
+                <option value="gcode">{{ _('Send G-code') }}</option>
+                <option value="opnative">{{ _('OctoPrint pause') }}</option>
+            </select>
+            <span class="help-block">When filament out is detected shall we send G-code to the printer or use OctoPrints pause function.</span>
+        </div>
+        <div class="controls" data-bind="visible: settingsViewModel.settings.plugins.filamentsensorsimplified.cmd_action() == 'gcode'">
+            <br/>
             <span class="help-block">Which G-code will be sent to printer on filament runout.</span>
+            <input id="filamentsensorsimplified_settings_commandInput" type="text" class="input-large" data-bind="value: settingsViewModel.settings.plugins.filamentsensorsimplified.g_code, disable:printing">
         </div>
     </div>
 
+    <h3>{{ _('Test sensor') }}</h3>
     <div class="control-group">
         <div class="controls">
-            <strong id="filamentsensorsimplified_settings_testResult" data-bind="html: testSensorResult"></strong>
             <input type="button" class="btn btn-info" data-bind="click: testSensor, disable:printing" value="Test sensor">
+            <strong id="filamentsensorsimplified_settings_testResult" data-bind="html: testSensorResult"></strong>
```

Comment:
Don't remember, was this test result text offset to side like this before? I think it was under the button.
![image](https://user-images.githubusercontent.com/13912644/131742974-b4a6493c-e9a1-400e-a39d-875424ebc293.png)